setup.py

```diff
@@ -5,6 +5,20 @@
 from subprocess import call
 from sys import platform, version_info
 
+# tentative requirements
+requires = ["argparse", "bs4", "pexpect", "requests", "backports.shutil_which", "six", "termcolor", "submit50>=2.4.5"]
+
+# check whether we have a Homebrew or Python installation of Python
+INSTALL_CERTS = "/Applications/Python 3.6/Install Certificates.command"
+HOMEBREW_PYTHON = "/usr/local/bin/python3"
+
+if platform == "darwin" and version_info >= (3, 6):
+    if not isfile(INSTALL_CERTS) or call(INSTALL_CERTS) != 0:
+        if not isfile(HOMEBREW_PYTHON):
```

Comment:
Checking for whether `python3` (via Homebrew) is installed in a specific path does feel hackish, though, and also a proxy for what we're really checking for. How do we ask in code the question "are the requisite certificates already installed, as would be the case if Homebrew installed Python"? We an presumably do all this with a `try/except`, detecting whatever sort of action (HTTPS connections?) would fail without this dependency?

Reply:
@dmalan sounds wise, I started Googling for such earlier as well. I'll dig around to find out and then push an update when I figure it out!